src/test/java/tests/EventTests.java

```diff
@@ -60,4 +60,15 @@ public void basics() {
     private final EventHandler fail2 = (IssueSelectedEventHandler) e -> fail("IssueSelectedEventHandler failed");
     private final EventHandler succeed1 = (BoardSavedEventHandler) e -> assertTrue(true);
     private final EventHandler fail1 = (BoardSavedEventHandler) e -> fail("BoardSavedEventHandler failed");
+
+    @Test
+    public void testSuperclassHandlerOnSubclassEvent() {
+        events.register(superclassHandler);
+
+        RepoOpenedEvent subclassEvent = new RepoOpenedEvent("");
+
+        events.post(subclassEvent);
+    }
+
+    private final EventHandler superclassHandler = (OpenReposChangedEventHandler) e -> assertTrue(true);
```

Comment:
These fields should be locals, since they aren't being shared. Also the `EventBus` instance probably shouldn't be shared, as it's possible for events to interfere.

We want to ensure that the subclass handler also fires, so I guess we should have both handlers:

- Dispatch superclass event, ensure subclass doesn't fire.
- Dispatch subclass event, sure both fire.